example/spec/parallel/ingestGranule/IngestUMMGSuccessSpec.js

```diff
@@ -245,14 +259,14 @@ describe('The S3 Ingest Granules workflow configured to ingest UMM-G', () => {
 
       destinations = [{
         regex: '.*.hdf$',
-        bucket: config.bucket,
+        bucket: config.buckets.protected.name,
```

Comment:
If we move the file to config.bucket, it's not in the protected or public bucket, and won't have a URL associated with it (because it's in the `internal` bucket)